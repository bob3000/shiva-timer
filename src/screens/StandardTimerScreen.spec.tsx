import * as React from 'react';
import * as renderer from 'react-test-renderer';
import StandardTimerScreen from './StandardTimerScreen';

jest.useFakeTimers();

describe('StandardTimerScreen', () => {
  it('snapshot test', () => {
    const tree = renderer.create(<StandardTimerScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
