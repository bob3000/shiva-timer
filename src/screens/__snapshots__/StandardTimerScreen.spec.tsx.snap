// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StandardTimerScreen snapshot test 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "height": "100%",
        "opacity": 0,
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignSelf": "flex-end",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 60,
          "borderWidth": 1,
          "color": "#000000",
          "marginRight": 5,
          "marginTop": 30,
          "opacity": 0.8,
          "paddingHorizontal": 15,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 30,
              "fontWeight": "bold",
            }
          }
        >
          â‹®
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "borderColor": "#000000",
          "borderRadius": 40,
          "borderWidth": 1,
          "marginTop": 10,
          "opacity": 0.8,
          "width": 280,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "marginTop": 50,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Array [
              Object {
                "backgroundColor": "#FFFFFF",
                "opacity": 0.8,
              },
              Object {
                "alignItems": "center",
                "borderColor": "#000000",
                "borderRadius": 100,
                "borderWidth": 1,
                "height": 200,
                "justifyContent": "center",
                "opacity": 0.8,
                "width": 200,
              },
            ]
          }
        >
          <Image
            source={1}
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                Object {
                  "height": undefined,
                  "width": undefined,
                },
                undefined,
              ]
            }
          />
          <View
            style={
              Object {
                "borderRadius": 40,
                "opacity": 1,
                "padding": 5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000000",
                  "fontSize": 50,
                  "fontWeight": "300",
                }
              }
            >
              Start
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "bottom": 0,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "padding": 10,
          "position": "absolute",
          "width": "100%",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Array [
              Object {
                "backgroundColor": "#FFFFFF",
                "opacity": 0.8,
              },
              Object {
                "alignItems": "center",
                "borderColor": "#000000",
                "borderRadius": 100,
                "borderWidth": 1,
                "height": 60,
                "justifyContent": "center",
                "opacity": 0.8,
                "width": 60,
              },
            ]
          }
        >
          <Image
            source={1}
            style={
              Array [
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
                Object {
                  "height": undefined,
                  "width": undefined,
                },
                undefined,
              ]
            }
          />
          <View
            style={
              Object {
                "borderRadius": 40,
                "opacity": 1,
                "padding": 5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#000000",
                  "fontSize": 15,
                  "fontWeight": "300",
                }
              }
            >
              Reset
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
